Gulpfile.mjs

```diff
@@ -521,7 +521,7 @@ function buildRollup(packages, buildStandalone) {
             // We have manually applied commonjs-esm interop to the source
             // for library not in this monorepo
             // https://github.com/babel/babel/pull/12795
-            if (!id.startsWith("@babel/")) return "compat";
+            if (!id.startsWith("@babel/")) return "default";
```

Comment:
Could you briefly explain why archived syntax plugins are using `compat` while other dependencies are using `default`? Intuitively we should apply the same interop for everything not in the monorepo.

Reply:
Docs: https://rollupjs.org/configuration-options/#output-interop

`default` matches what Node.js does, so it's what we should use for external packages. For syntax plugins it doesn't really matter, because they all have `.default` and `.__esModule` so the `compat` and `default` work the same. But yeah, lets use `default` everywhere.